CONTRIBUTING.md

```diff
@@ -29,8 +29,214 @@ used wradlib as well as Python and OS version, error messages and instructions t
 
 ## Forking wradlib
 
-For this purpose, you need to have a [GitHub Account](https://github.com/signup/free) and [Git](https://git-scm.com/). Then fork wradlib, make changes and create a pull request. A [step-by-step guide](http://docs.wradlib.org/en/latest/community.html#how-to-contribute-to) can be found in the docs.
-
-## Development
-
-To keep the code as clean as much all code contributions should be in line with [PEP 8](https://www.python.org/dev/peps/pep-0008/) conventions.
+For this purpose, you need to have a [GitHub Account](https://github.com/signup/free) and [Git](https://git-scm.com/). Then fork wradlib, make changes and create a pull request.
+
+## Guide
+
+This is a practical step by step guide. The commands should work in your operating system terminal. Please adapt it to your needs.
+
+### 1a Install development environement using conda
+
+- Install [Anaconda or Miniconda](https://www.anaconda.com/download/)
```

Comment:
Maybe just refer to the general `Installation`-section for the conda based install.

Reply:
I actually propose a minimal working installation of conda. Please check the new specific guide for windows(the linux guide is coming). The installation of dev and opt dependencies remain an open question. A reference to the more elaborated installation guide can be made for better context.